internal/interpreter/eval.rs

```diff
@@ -606,6 +607,12 @@ fn call_builtin_function(
                     parent_item_info.item_index(),
                 );
 
+                let close_policy = match popup.close_policy.value {
+                    0 => ClosePolicy::OnClick,
+                    1 => ClosePolicy::OnClickOutside,
+                    _ => ClosePolicy::NoAutoClose,
+                };
+
```

Comment:
When we add new variants to the enum, we're likely to forget adjusting this place. It would be better to go the slower but safer route of automatic conversion:

```suggestion
                let close_policy = Value::EnumerationValue(
                    popup.close_policy.enumeration.name.to_string(),
                    popup.close_policy.to_string(),
                )
                .try_into()
                .expect("Invalid internal enumeration representation for close policy");

```